SELF-ASSESMENT.md

```diff
@@ -44,3 +56,359 @@ See [README](https://github.com/jaegertracing/jaeger/#jaeger---a-distributed-tra
+
+### [CII Best Practices](https://www.coreinfrastructure.org/programs/best-practices-program/).
+
+_Best Practices. A brief discussion of where the project is at with respect to CII best practices and what it would need to achieve the badge._
+
+* The Jaeger project has achieved passing level criteria under Open Source Security Foundation Best Practices and is in the process of obtaining the silver badge.
+* Jaeger-client-js and Jaeger-client-javascript is in the process of obtaining passing level criteria.
+
+### Case Studies.
+
+_Provide context for reviewers by detailing 2-3 scenarios of real-world use cases._
+
+#### Ticketmaster:
+
+* Consisting of over 300 microservices, the company benefits from the Jaeger project, utilizing the system to track over 100 million transactions a day. 
+* As the company scaled, they quickly learned that logging was insufficient and bulky. It helped uncomplicate the log aggregation process which generated terabytes of data daily. Jaeger simplified the debugging process by providing visibility into critical areas. – where developers could find root causes of issues and save time doing so.
+* Ticketmaster uses the Directed Acyclic Graph (DAG) view to see request flow. 
+* They also take advantage of Jaeger’s powerful adaptive sampling feature which reduces overhead without reducing visibility into systems.
+
+#### Grafana Labs:
+
+* They needed a solution to trace request issues end-to-end. Grafana sometimes dealt with high request latency and opaque request paths. 
+* Jaeger allowed engineers to sort requests by duration with the Jaeger UI, which identified slow problematic requests and allowed for more granular troubleshooting. 
+* The immediate effect of this higher efficiency troubleshooting was the ability to increase query optimization, sometimes by 5 to 10x. 
+* One feature they particularly like is the contextual logging capability.
+
+#### Logz.io:
+
+* The company was dealing with seemingly random decreasing performance issues in certain request flows. Logs failed to provide the answer or clue they were looking for. 
+* They incorporated Jaeger into their Node.js HTTP layers. 
+* The Jaeger UI immediately identified a series of issues including the serial call (a.k.a “staircase”) and a “span spike” (a request that ranged from a few milliseconds to 10 seconds).
+* They correlate logs with traces. These combined allow for a powerful troubleshooting system. Logz.io does so by attaching a request ID field to every log and adding it as a tag to the respective span.
+
+### Related Projects / Vendors.
```

Comment:
Please remove this. I don't want to maintain comparative analysis with competitors - things change all the time. I don't even want to maintain a list of possible competitors - why should we advertise them on our site?